ext/bcmath/tests/str2num_formatting.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+1 and 2 argument of bcadd/bcsub/bcmul/bcdiv/bcmod/bcpowmod/bcpow/bccomp (last one works different then others internally);
+1 argument of bcsqrt - must be well-formed
```

Comment:
`--TEST--` should be just a single line -- for longer description `--DESCRIPTION--` can be used, or a comment in the code.